src/App/App.js

```diff
@@ -2,15 +2,15 @@ import React, { Component } from 'react';
 import Navbar from '../Navbar/Navbar';
 import Movie from '../Movie/Movie';
 import AllMovies from '../All-Movies/All-Movies';
-import movieData from '../movieData';
 import { getMovie } from '../api-calls/apiCalls'
+import { getAllMovies } from '../api-calls/apiCalls'
```

Comment:
I think this may be able to be written `{ getMovies, getAllMovies }` so it is exported as one object, then it can be destructured as `{ getMovies, getAllMovies }` in the `App` import. It would work the same, so maybe something to think about as a clean-up, but not necessary.